### PR TITLE
Remove footer field from theme submission form

### LINKED ISSUE #5379: Remove footer field from theme submission form
### Describe the problem and steps to reproduce it:
Theme footers are obsolete, so we should remove the field on the theme submission form where we ask for a footer image.

### Anything else we should know?
https://www.dropbox.com/s/k5gvodd0c88pazg/Screenshot%202017-05-10%2011.31.23.png?dl=0

### PR DESCRIPTION
**Pull request by atiqueahmedziad**
Fixes #5379

### CODE REVIEW

**Comment by atiqueahmedziad** [comment]
@EnTeQuAk  Can you help me to fix the errors with the latest commit?

**Comment by EnTeQuAk** [comment]
Sorry, I didn't get to this PR before my PTO last week. I'll look into it shortly.

File: src/olympia/devhub/tests/test_views_submit.py [hunk 1]
```diff
@@ -39,21 +39,19 @@ def setUp(self):
 
     def get_img_urls(self):
         return (
-            reverse('devhub.personas.upload_persona', args=['persona_header']),
-            reverse('devhub.personas.upload_persona', args=['persona_footer'])
+            reverse('devhub.personas.upload_persona', args=['persona_header'])
```
Comment (EnTeQuAk) [inline src/olympia/devhub/tests/test_views_submit.py]: This isn't a tuple anymore, if you add a `,` at the end it's a tuple again and the tests should work again.

```python
return (
    reverse('devhub.personas.upload_persona', args=['persona_header']),
)
```

**Comment by EnTeQuAk** [comment]
Hey @atiqueahmedziad did you have time to look into the test failures and the comment?

I'm going to close the pull request for now but that's just to clean-up our queue a bit, if you have the time to look into updating it please feel free to re-open it any time so we can get that change into addons-server.

**Pull request closed without merging**